packages/xchain-thorchain/package.json

```diff
@@ -27,9 +27,11 @@
     "clean": "rimraf lib/**",
     "build": "yarn clean && rollup -c",
     "test": "jest",
+    "e2e": "jest --config jest.config.e2e.js",
     "lint": "eslint \"{src,__tests__}/**/*.ts\" --fix --max-warnings 0",
     "prepublishOnly": "yarn build",
-    "start:example": "ts-node example/index.ts"
+    "start:example": "ts-node example/index.ts",
+    "generate:MsgDeposit": "./genMsgDeposit.sh"
```

Comment:
Pls put a note about this command into the README.

Reply:
I added a note here: https://github.com/xchainjs/xchainjs-lib/pull/501/files#diff-c903bffebd5e89b3f2f3b6fc7813460bc9b3edd1dd386262631c0732712b4cf9R68 - let me know if you want something else

Reply:
Note looks good! But I could not find anything in the README about `generate:MsgDeposit` command ...

Reply:
added